### DISCUSSION TITLE
Set<*>.containsExactly

### DISCUSSION THREAD

**Discussion by oldergod** [assertk-org/assertk#489] [General]
The assertion `containsExactly` doesn't exist for `Set`s. Is there a reason to this? One has to transform the type to use either the list or the array assertion.

**Comment by evant** [comment]
It's because `containsExactly` cares about order and `Set`s don't have a defined order. You can use `containsOnly` if you don't care about ordering or convert it to a list to define the order.

**Reply by oldergod** [reply]
Oh interesting. I forgot about it because assertj offers this method. Thank you then!
